src/lib/openapi/spec/client-feature-schema.ts

```diff
@@ -9,55 +9,67 @@ export const clientFeatureSchema = {
     $id: '#/components/schemas/clientFeatureSchema',
     type: 'object',
     required: ['name', 'enabled'],
+    description:
+        'Feature toggle configuration used by SDKs to evaluate state of a toggle',
     additionalProperties: false,
     properties: {
         name: {
             type: 'string',
+            description:
+                'The unique name of a feature toggle. Is validated to be URL safe on creation',
+            example: 'new.payment.flow.stripe',
         },
         type: {
             type: 'string',
+            description: 'What kind of feature flag is this',
+            example: 'release',
         },
         description: {
             type: 'string',
+            description: 'A description of the toggle',
             nullable: true,
-        },
-        createdAt: {
-            type: 'string',
-            format: 'date-time',
-            nullable: true,
-        },
-        lastSeenAt: {
-            type: 'string',
-            format: 'date-time',
-            nullable: true,
+            example: 'No variants here',
```

Comment:
While this is funny, I'm afraid it could also be misleading. I'd suggest something (sadly) more mundane. How about this? It links together with the feature name example

```suggestion
            example: 'Enable the new payment flow',
```